Lekce_02_Ukol04/Lucistnik.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace Lekce_02_Ukol04
+{
+    internal class Lucistnik
+    {
+        private int PocetSipu;
+
+        public Lucistnik(int pocetSipu)
+        {
+            if (pocetSipu >= 0)
+            {
+                PocetSipu = pocetSipu;
+            }
+            else
+            {
+                Console.WriteLine("Počet šípů nemůže být negativní.");
+            }
+        }
+
+        public void Vystrel()
+        {
+            PocetSipu--;
```

Comment:
Meli bychom odebrat pocet sipu pouze pokud pocet sipu neni mensi nebo rovno nule. Kdybys zavolala tuhle metodu, zapise se do konzole `Nemam sipy` ale porad odeberes pocet sipu o jedna, takze to muze jit do negatvniho cisla (pokud to neosetri nekdo zvenci jako tomu je v `Program.cs`

Reply:
opraveno